bagel/cli.py

```diff
@@ -1,22 +1,16 @@
-import json
 from pathlib import Path
 
 import typer
 from bids import BIDSLayout
 
-import bagel.bids_utils as butil
-import bagel.derivatives_utils as dutil
-import bagel.file_utils as futil
-import bagel.pheno_utils as putil
 from bagel import mappings, models
-from bagel.derivatives_utils import PROC_STATUS_COLS
-from bagel.utility import (
-    confirm_subs_match_pheno_data,
-    extract_and_validate_jsonld_dataset,
-    generate_context,
-    get_imaging_session_instances,
-    get_subject_instances,
-)
+
+from .utilities import bids_utils as butil
+from .utilities import derivatives_utils as dutil
+from .utilities import file_utils as futil
+from .utilities import model_utils as mutil
+from .utilities import pheno_utils as putil
```

Comment:
:cherries: is that necessary? I find `file_utils` clearer to understand than `futil` etc as it also matches the module name. Might make the code a little longer though ...

Reply:
alternative idea: package `utilities` -> `util` and then you can just do `import util` and refer to `util.file`, `util.bids`, `util.model`. almost as short, but still clear